### ISSUE TITLE
Unknown variable: '::lsbdistcodenameid' puppet 4.5.1 -  Debian wheezy

### ISSUE THREAD

**Issue by reinaldomendes** [mayflower/puppet-php#196]
I need to disable strict variables on puppet.conf because  I receive a message "Unknown variable: '::lsbdistcodenameid'. at php/manifests/params.pp:38:27"

When I run I receive many warnings

```bash


Warning: Unknown variable: '::lsbdistcodenameid'. at /tmp/vagrant-puppet/environments/modules/php/manifests/params.pp:38:27
Warning: Scope(Apt::Key[php::repo::debian]): $key is deprecated and will be removed in the next major release. Please use $id instead.
Warning: Scope(Apt::Key[php::repo::debian]): $key_source is deprecated and will be removed in the next major release. Please use $source instead.
Warning: Scope(Apt::Source[source_php_wheezy-php55]): $include_src is deprecated and will be removed in the next major release, please use $include => { 'src' => false } instead
Warning: Scope(Apt::Source[dotdeb-wheezy]): $include_src is deprecated and will be removed in the next major release, please use $include => { 'src' => false } instead

```

**Comment by mmoll** [comment]
this already got fixed in ba6cb18

**Comment by mmoll** [comment]
Regarding the apt warnings, I'll re-check that after #190 got in, we probably have to bump our puppetlabs-apt dependency to 2.x and update our includes of it.
